client/network_status.go

```diff
@@ -161,3 +162,37 @@ func getSiteNameMap(sites *[]types.SiteInfo) map[string]string {
 
 	return siteNameMap
 }
+
+func (cli *VanClient) GetRemoteLinks(ctx context.Context, siteConfig *types.SiteConfig) ([]*types.RemoteLinkInfo, error) {
+
+	//Checking if the router has been deployed
+	_, err := cli.KubeClient.AppsV1().Deployments(cli.Namespace).Get(types.TransportDeploymentName, metav1.GetOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("skupper is not installed: %s", err)
+	}
+
+	currentSiteId := siteConfig.Reference.UID
+
+	sites, err := server.GetSiteInfo(cli.Namespace, cli.KubeClient, cli.RestConfig)
```

Comment:
This call to `GetSiteInfo` is never completing when a link is established between remote sites (in my
case, between minikube / ibmcloud).

Digging into `GetSiteInfo` it is performing a `try.Until` with a  timeout of 3 seconds only, which is never
enough. Maybe the same `timeout` that has been added to `link status` could be passed through to
`GetSiteInfo` as well, so it has enough time to complete.

Reply:
Absolutely, thanks!